### PR TITLE
added destructuring

### PR DESCRIPTION
**Pull request by vigneshwaran-chandrasekaran**

### CODE REVIEW

File: .prettierrc.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// prettier.config.js or .prettierrc.js
+module.exports = {
+	trailingComma: 'es5',
+	tabWidth: 4,
+	useTabs: true,
+	semi: true,
+	singleQuote: true,
+	'editor.formatOnSave': true,
+	// printWidth: 80,
+	proseWrap: 'always',
```
Comment (GermaVinsmoke) [inline .prettierrc.js:10]: What's the use of `proseWrap: 'always'`? I've never used this before. I'll also learn something from this 🙂

File: src/components/BmiForm/BmiForm.jsx [hunk 2]
```diff
@@ -1,33 +1,32 @@
 import React, { useState } from 'react';
-import '../App/App.css';
 import PropTypes from 'prop-types';
+import '../App/App.css';
```
Comment (GermaVinsmoke) [inline src/components/BmiForm/BmiForm.jsx:3]: This is good, CSS imports should be at the bottom. I should've corrected this mistake of mine 🙄

File: .prettierrc.js (hunk 1 above)
Comment (vigneshwaran-chandrasekaran) [reply .prettierrc.js:10]: https://prettier.io/docs/en/options.html#prose-wrap

Comment (GermaVinsmoke) [reply .prettierrc.js:10]: Thanks 🙂

**Pull request merged**